.github/CONTRIBUTING.md

```diff
@@ -1 +1,253 @@
-Coming soon
\ No newline at end of file
+# Contributing Guidelines
+
+Welcome to the project, hope you enjoy your stay. The guidelines below will detail what you need to know in order to be a productive contributor as well as a few standards for keeping everyone's contributions consistent.
+
+## Table of Contents
+
+[**What should I know before I get started?**](#what-should-i-know-before-i-get-started)
+
+* [ES6](#ecmascript-6)
+* [React](#react-fe)
+* [Redux](#redux-fe)
+* [Express](#express-be)
+* [Socket.io](#socketio-be)
+* [Cypress.io](#cypressio-testing)
+
+[**Where to Start**](#where-to-start)
+
+* [This Document](#this-document)
+* [Existing Issues](#existing-issues)
+* [Your Ideas](#your-ideas)
+
+[**Style Guides**](#style-guides)
+
+* [Wireframes](#wireframes)
+* [Case Standards and Naming Conventions](#case-standards-and-naming-conventions)
+* [JavaScript Style Guide](#javascript-style-guide)
+* [Documentation and Comments](#documentation-and-comments)
+* [Issues and Pull Requests](#issues-and-pull-requests)
+* [Git Branches and Commits](#git-branches-and-commits)
+
+[**Design Decisions**](#design-decisions)
+
+* [Why React and Redux](#why-react-and-redux)
+* [Handling Side Effects](#handling-side-effects)
+* [Emitting Updates to Clients](#emitting-updates-to-clients)
+* [Choosing Component Types](#choosing-component-types)
+* [API Naming Convention](#api-naming-convention)
+
+[**Priorities**](#priorities)
+
+* [Size Matters](#size-matters)
+* [Minimal Renders](#minimal-renders)
+
+-----
+
+## What should I know before I get started?
+
+The following section gives a brief explanation of how each library fits into the project as well as links to starting resources. You don't need to be familiar with all the libraries in order to contribute but it is recommended you familiarise yourself with those relevant to whichever part of the project you want to get involved with.
```

Comment:
Might also be worth noting the `good first issue` tag here too? I'm always happy to go through learning resources but sometimes would prefer to do so while working on something practical. 

I just wonder if some may be put off by the perceived amount of learning they may need to do before getting started as a contributor. Having to scan through all this before seeing anything regarding that tag means people may not even get to that point.